11/main.js

```diff
@@ -0,0 +1,294 @@
+      certificate: true,
+      dateOfExpiry: "21/10/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "USA",
+    productPrice: 211,
+  },
+  
+];
+let i = 0;
+const div = document.createElement("div");
+div.className = "box";
+document.body.append(div);
+function createListAndInsertToHTML() {
+  const ul = document.createElement("ul");
+  div.append(ul);
+  for (let key of listOfProducts) {
+    const li = document.createElement("li");
+    li.innerHTML = key.productName;
+    i--;
+    ul.append(li);
+    const childUl = document.createElement("ul");
+    li.append(childUl);
+    for (let parameter in key) {
+      i++;
+      if (parameter !== "productName" && !(key[parameter] instanceof Object)) {
+        const childLi = document.createElement("li");
+        childLi.innerHTML = `${parameter} : ${key[parameter]}`;
+        isElementTwin(childLi, i);
+        childUl.append(childLi);
+        if (parameter == "productCountry") {
+          const image = document.createElement("img");
+          if (key[parameter] == "UA") {
+            IMGSetAttribute(image, UaIMGUrl);
+            childLi.append(image);
+          }
+          if (key[parameter] == "russia") {
+            IMGSetAttribute(image, russiaIMGUrl);
+            childLi.append(image);
+          }
+          if (key[parameter] == "USA") {
+            IMGSetAttribute(image, USAIMGRUrl);
+            childLi.append(image);
+          }
+          if (key[parameter] == "UK") {
+            IMGSetAttribute(image, UKIMG);
+            childLi.append(image);
+          }
+          if (key[parameter] == "Italia") {
+            IMGSetAttribute(image, ItaliaIMG);
+            childLi.append(image);
+          }
+          if (key[parameter] == "Francia") {
+            IMGSetAttribute(image, FranciaIMG);
+            childLi.append(image);
+          }
+        }
+        if (parameter == "productPrice"){
+          totalsum += key[parameter]
+          allProductPrices.push(key[parameter])
+        }
+      } else if (key[parameter] instanceof Object) {
+        const liOfObject = document.createElement("li");
+        liOfObject.innerHTML = `${parameter}`;
+        isElementTwin(liOfObject, i);
+        childUl.append(liOfObject);
+        const childObjectUl = document.createElement("ul");
+        liOfObject.append(childObjectUl);
+        for (let newkey in key[parameter]) {
+          i++;
+          const childLiOfObject = document.createElement("li");
+          childLiOfObject.innerHTML = ` ${newkey} : ${key[parameter][newkey]}`;
+          if (newkey== 'certificate'){
+            if (key[parameter][newkey] === false){
+              childLiOfObject.setAttribute('id', "attention")
+            }
+          }
+          isElementTwin(childLiOfObject, i);
+          childObjectUl.append(childLiOfObject);
+        }
+      }
+    }
+  }
+}
+
+createListAndInsertToHTML();
+function isElementTwin(sometag, pointer) {
```

Comment:
Try to initialize function before usage

```suggestion
function isElementTwin(someTag, pointer) {
```